Update IP address information for availability (#76225)
Azure/azure-content: articles/azure-monitor/app/ip-addresses.md

Open ports 80 (http) and 443 (https) for incoming traffic from these addresses (IP addresses are grouped by location):

### IP Addresses

If you're looking for the actual IP addresses so you can add them to the list of allowed IP's in your firewall, please download the JSON file describing Azure IP Ranges. These files contain the most up-to-date information.

After downloading the appropriate file, open it using your favorite text editor and search for "ApplicationInsightsAvailability" to go straight to the section of the file describing the service tag for availability tests.

> [!NOTE]
> These addresses are listed using Classless Inter-Domain Routing (CIDR) notation. This means that an entry like `51.144.56.112/28` is equivalent to 16 IPs starting at `51.144.56.112` and ending at `51.144.56.127`.

#### Azure Public Cloud

> Download from [here](https://www.microsoft.com/en-us/download/details.aspx?id=56519).

Reviewer: ```suggestion
Download [Public Cloud IP addresses](https://www.microsoft.com/download/details.aspx?id=56519).
```

Author: Makes sense! Thanks!

#### Azure US Government Cloud

> Download from [here](https://www.microsoft.com/en-us/download/details.aspx?id=57063).

Reviewer: ```suggestion
Download [Government Cloud IP addresses](https://www.microsoft.com/download/details.aspx?id=57063).
```

Author: Makes sense! Thanks!

#### Azure China Cloud

> Download from [here](https://www.microsoft.com/en-us/download/details.aspx?id=57062).

Reviewer: ```suggestion
Download [China Cloud IP addresses](https://www.microsoft.com/download/details.aspx?id=57062).
```

Author: Makes sense! Thanks!

### Discovery API

> You may also want to [programmatically retrieve](https://docs.microsoft.com/en-us/azure/virtual-network/service-tags-overview#use-the-service-tag-discovery-api-public-preview) the current list of service tags together with IP address range details.

Reviewer: ```suggestion
You may also want to [programmatically retrieve](../../virtual-network/service-tags-overview.md#use-the-service-tag-discovery-api-public-preview) the current list of service tags together with IP address range details.
```

Author: Makes sense! Thanks!
